dev-php/pecl-ast/pecl-ast-1.0.5.ebuild

```diff
@@ -0,0 +1,14 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+USE_PHP="php7-1 php7-2 php7-3 php7-4"
```

Comment:
php-7.1 is masked, you shouldn't add support for it in new packages. It's getting removed in 30 days.